### PR TITLE
Document pr self-approve

### PR DESCRIPTION
**Pull request by yutongz**
**Release note**:

```release-note
None
```

### CODE REVIEW

**Review by gyliu513 — COMMENTED** [review]
/lgtm

File: mungegithub/deployment/README.md [hunk 1]
```diff
@@ -26,6 +26,8 @@ Mungegithub waits for 4 labels:
 
   >  Note: Do not comment "/approve" or add "approved" label unless you are 100% sure you want this change, because after you say that, the pr will be merge any minutes.
   
+  >  Note: The PR will be self-approved if the PR creator is in OWNERS file and this PR is associated with at lease one issue.
```
Comment (nclando) [inline mungegithub/deployment/README.md]: with at least*

**Comment by yutongz** [comment]
@nlandolfi  PTAL

**Review by sebastienvas — APPROVED** [review]

**Pull request merged**